tests/acceptance/subscribe/SubscribeOnOrderPendingPaymentEventCest.php

```diff
@@ -85,9 +85,6 @@ public function testOptInWhenUncheckedWithFormAndSimpleProduct(AcceptanceTester
 
 		// Confirm that the email address was not added to ConvertKit.
 		$I->apiCheckSubscriberDoesNotExist($I, $result['email_address']);
-
-		// Unsubscribe the email address, so we restore the account back to its previous state.
-		$I->apiUnsubscribe($result['email_address']);
```

Comment:
Similarly, there are quite a few places where a call is made to unsubscribe an email address after checking a subscriber does not exist. This didn't make sense, because there's no subscriber to unsubscribe...